pkg/sentry/options.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
This file was copied almost as-is from components-contrib. We may need to move it to dapr/kit eventually

Reply:
I think moving to `/kit` now makes sense.